pypeit/fluxspec.py

```diff
@@ -444,22 +447,27 @@ def generate_sensfunc(self):
             return None
 
         ext_final = fits.getheader(self.par['std_file'], -1)
-        norder = ext_final['ECHORDER'] + 1
+        #norder = ext_final['ECHORDER'] + 1
+        norder = len(self.std)
```

Comment:
this might be tricky if you have two objects in the standard 1d spectra file

Reply:
My change ignores the spectra file.
It keys on `self.std` which has been constructed
to be only a single object.   Do you agree?

Reply:
okay. I forgot that we have already run the find_standard before this step.